### ISSUE TITLE
GetConditionCodes is missing from the library

### ISSUE THREAD

**Issue by gcatlin** [alpacahq/alpaca-trade-api-go#287]
https://docs.alpaca.markets/reference/stockmetaconditions-1

**Comment by ankitsridhar16** [comment]
is the requirement for the ticket is to implement this rest route ?

**Comment by gcatlin** [comment]
Yes, exactly. It also entails creating a new `Conditions` entity, `ConditionType` enum, and `Tape` enum. The response body looks like this:
```
```

**Comment by ankitsridhar16** [comment]
Okay you can assign me the issue I would like to work on this

**Comment by gcatlin** [comment]
I don't have access to assign it to you, but it's safe for you to start working on it. Thanks for taking these issues on!

**Comment by zrhmn** [comment]
Hi. It seems like this has not yet been addressed. I've been looking to make contributions, if @ankitsridhar16 does not mind, I can take this over and have a PR done soon.
